sdk/js/src/Request.ts

```diff
@@ -94,6 +110,28 @@ export const publishEvents = async (
     return res
 }
 
+export const saveEntity = async (user: DVCUser, envKey: string): Promise<AxiosResponse> => {
+    if (!envKey) {
+        throw new Error('Missing envKey to save to cloud data!')
+    }
+
+    if (!user || !user.user_id) {
+        throw new Error('Missing user to save to cloud data!')
+    }
+
+    const res = await patch(
+        `${BASE_URL}${HOST}${SAVE_ENTITY_PATH}`,
+        envKey,
+        user as unknown as Record<string, unknown>
+    )
+    if (res.status !== 201) {
```

Comment:
is 201 the only possible success code here? Isn't 200 also valid?

Reply:
oop copied it from the publish events endpoint. Does the same apply for the publish events endpoint? should we just look for responses under some number (maybe <400)?